.github/workflows/chromatic.yml

```
@@ -0,0 +1,53 @@
+      - name: Build Storybook
+        run: pnpm run build-storybook
+
+      - name: Publish to Chromatic
+        id: chromatic
+        uses: chromaui/action@v1
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          storybookBuildDir: storybook-static
+          comment: true
+          commentTag: chromatic-preview
+          message: |
```

actionlint (1.7.7)

property "chromatic" is not defined in object type {}

(expression)